pkg/api/folder_permission.go

```diff
@@ -153,10 +153,10 @@ func (hs *HTTPServer) getFolderACL(ctx context.Context, user identity.Requester,
 			FolderUID:      folder.ParentUID,
 			Created:        p.Created,
 			Updated:        p.Updated,
-			UserID:         p.UserId,
+			UserID:         p.UserID,
 			UserLogin:      p.UserLogin,
 			UserEmail:      p.UserEmail,
-			TeamID:         p.TeamId,
```

Comment:
nit: why is there no teamUID here? same with the user UID

it's not part of the resource permissions, so this is fine